modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -315,12 +315,45 @@ definition {
 	}
 
 	@description = "Verify that the user can order the results inside Assigned to Me by Due Date"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOrderAssignedtoMeTasksByDueDate {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.updateDueDateTaskByActions(
+			workflowAssetTitle = "Blogs Entry",
+			workflowAssetType = "Blogs Entry",
+			workflowDueDateTime = "1:45 PM",
+			workflowTask = "Review");
+
+		Workflow.assignToMeTaskByDetails(
+			workflowAssetTitle = "Blogs Entry",
+			workflowAssetType = "Blogs Entry",
+			workflowTask = "Review");
 
-		// TODO LRQA-68849 CanOrderAssignedtoMeTasksByDueDate pending implementation
+		Notifications.goToFilterValues(filterValues = "Due Date");
 
+		AssertTextEquals.assertPartialText(
+			key_workflowAssetTitle = "Blogs Entry",
+			key_workflowTask = "Review",
+			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_DUE_DATE",
+			value1 = "Within 8 Months");
```

Comment:
Try to create macros from these functions when you feel the need to use them in .testcase files. Complementing the comment above, ideal is to use a macro to ensure the position of items, you can replace this piece and use the same macro viewMyWorkflowTaskItemPosition and path MY_WORKFLOW_TASK_ITEM_POSITION that you have created at PR #843. Thank you!

Reply:
You can take your own [PR](https://github.com/rodrigocunhaa/liferay-portal/pull/837) as example to what you can do with that one.